### PR TITLE
UI added from Yansen

### PR DESCRIPTION
**Pull request by liux1731**

### CODE REVIEW

File: project/lang/.htaccess [hunk 1]
```diff
@@ -0,0 +1 @@
+deny from all
```
Comment (buse2017fall) [inline project/lang/.htaccess:1]: Why we need to change this file

**Review by buse2017fall — CHANGES_REQUESTED** [review]

File: project/lang/.htaccess (hunk 1 above)
Comment (liux1731) [reply project/lang/.htaccess:1]: add language setting

File: project/lang/class.default.php [hunk 2]
```diff
@@ -0,0 +1,198 @@
+<?php
```
Comment (buse2017fall) [inline project/lang/class.default.php:1]: What is the purpose of this file ?

**Review by buse2017fall — CHANGES_REQUESTED** [review]

**Review by buse2017fall — APPROVED** [review]
Ship it !

**Pull request merged**